healthcheck

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -euo pipefail
+
+if [ $# -ne 2 ]; then
+    echo "Usage: $0 HOST PORT
+
+Healthcheck for the RT docker container."
+    exit 1
+fi
+
+host="$1"
+port="$2"
+
+resp=$(curl -s -d @- -H "Host: rt.ocf.berkeley.edu" \
+       "http://$host:$port/REST/1.0/ticket/1/view" <<EOF
```

Comment:
Should we continue to use the mesos agent's host and port (e.g. `jaws:31811`) or simply use `localhost` instead? The former is a little more end to end, the latter is a little simpler.

Reply:
I suggest leaving it like this because it doesn't hurt and it means we can run this script from another host if we want. As it is, the caller can choose to use either.